### PR TITLE
Update package.json for steal.js support

### PR DESCRIPTION
**Pull request by omerts**
In browser loaders are becoming popular, and the added config, adds support for importing react-router with steal.js (and systemjs)

### CODE REVIEW

**Comment by WRidder** [comment]
+1 for systemjs support

Op do 21 mei 2015 om 16:41 schreef omerts <notifications@github.com>:

**Comment by ryanflorence** [comment]
umd package moved, check out latest master, mind updating and I'll merge?

**Comment by omerts** [comment]
@ryanflorence  sorry for the late response. I have updated the file to point to the new path, and also to the minified file.

File: package.json [hunk 1]
```diff
@@ -65,5 +65,8 @@
     "route",
     "routes",
     "router"
-  ]
+  ],
+  "system": { 
+    "main": "lib/umd/ReactRouter.min"
```
Comment (mjackson) [inline package.json]: Are you sure you want to use the minified file, @omerts? Seems like bundlers would want the unminified file.

**Comment by omerts** [comment]
It is for when you install the package in stealjs, so the package manager knows what is the entry point, so  I guess for just using the package the minified file is better.

**Comment by matthewp** [comment]
Steal maintainer here, you want the non minified version.

**Comment by omerts** [comment]
@matthewp I'd be happy to know why?  I thought that for 3rd party libs mini field should be better.

**Comment by ryanflorence** [comment]
we've decided to only publish to npm, package.json should be enough for any loader.

**Pull request closed without merging**